Formula/libfido2.rb

```diff
@@ -0,0 +1,28 @@
+class Libfido2 < Formula
+  desc "Provides library functionality for FIDO 2.0, including USB connections"
+  homepage "https://developers.yubico.com/libfido2/"
+  url "https://github.com/Yubico/libfido2/archive/1.3.0.tar.gz"
+  sha256 "0b2e3671c4c5d42fd5604a08e45f89f49592b97cf66d7d3bfbc7e6a4d5a0fec7"
+  depends_on "cmake" => :build
+  depends_on "mandoc" => :build
+  depends_on "libcbor"
+  depends_on "openssl@1.1"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "man_symlink_html"
+      system "make", "install"
+    end
+    # TODO: This doesn't quite work for man3, since manual pages document multiple functions
+    # TODO: This doesn't solve the man/ directory appearing in the audit
+    man1.install Dir["man/*.1"]
+    man3.install Dir["man/*.3"]
+  end
+
+  # TODO: Needs more comprehensive tests
```

Comment:
If it's a library, just compile something with it.

Reply:
Agreed! I've cleaned up and pushed one of the tests that I've written, and I'm working on tests that exercise the cryptographic functionality.